### PR TITLE
Remove browse list component and custom styles from homepage

### PR DESCRIPTION
**Pull request by lfdebrux**
Ticket: https://trello.com/c/as2lcdbr/263-3-remove-govuktemplate-from-buyer-frontend

This PR refactors our homepage to use GOV.UK Frontend styles. This was done to make it simpler to complete #975, removing govuk_template.

The visual changes are minimal, with one important exception: all links are now underlined, including the main "top level" links. This change has been made because having links without underlines is a [WCAG 2.1 failure condition, "creating links that are not visually evident without color vision"](https://www.w3.org/WAI/WCAG21/Techniques/failures/F73.html).

#### Before

Digital Marketplace homepage as in production

#### After

Digital Marketplace homepage as in this pull request, with underlined links

### CODE REVIEW

File: app/templates/choose-lot.html [hunk 1]
```diff
@@ -24,7 +24,7 @@
 
 {% block mainContent %}
 
-<div class="index-page">
+<div>
```
Comment (aliuk2012) [inline app/templates/choose-lot.html]: Looking at the rest of the page structure, I think this div is unnecessary.

**Comment by katstevens** [comment]
How will this look with the temporary message for an open framework?

**Comment by lfdebrux** [comment]
> How will this look with the temporary message for an open framework?

It looks pretty much the same 😄 

### Screenshots

#### Before

Digital Marketplace homepage on preview, with temporary message card for coming framework G12

#### After

Digital Marketplace homepage with this change, with temporary message card for coming framework G12

File: app/templates/index.html [hunk 2]
```diff
@@ -16,100 +16,147 @@ <h1>
...
+          <strong>
+            <a class="govuk-link"
+               href="{{ url_for('external.info_page_for_starting_a_brief', framework_slug=dos_slug, lot_slug='digital-specialists') }}">
+              Find an individual specialist
+            </a>
+          </strong>
           <p class="govuk-body">
-            <a href="/digital-outcomes-and-specialists/opportunities" class="govuk-link top-level-link">
-              View Digital Outcomes and Specialists opportunities
+            eg a developer or user researcher
```
Comment (katstevens) [inline app/templates/index.html]: One day we will get rid of all these `eg`s. But not in this PR 😸

File: app/templates/index.html [hunk 3]
```diff
@@ -16,100 +16,147 @@ <h1>
...
+            {% include "toolkit/temporary-message.html" %}
+          {% endwith %}
+        </li>
+        {% endif %}
+
+        {% if current_user.role != 'supplier' %}
+        <li class="govuk-!-padding-bottom-6">
+          <strong>
+            <a class="govuk-link"
+               {# TODO: add an external link for this #}
```
Comment (katstevens) [inline app/templates/index.html]: I know this was here before - do we have a tech debt card for this?

Comment (lfdebrux) [reply app/templates/index.html]: There is not, I can create one

Comment (lfdebrux) [reply app/templates/index.html]: https://trello.com/c/5MAkI3PW/1237-set-up-process-for-recurrent-data-inventory-review

**Pull request merged**